package.json

```diff
@@ -42,7 +42,7 @@
     "sass-export": "^2.1.2",
     "slugify": "^1.6.5"
   },
-  "dependencies": {
-    "npm": "^10.5.2"
```

Comment:
NPM shouldn't be a runtime dependency of this project. We shell out to `npm`, but that uses the same global install you used to install the package, not the second copy this installs into node_modules. It is quite hard to deduce the intent of this from the commit it was added in.